### PR TITLE
[DevPack Compiler Optimize] fix push bool

### PR DESCRIPTION
**Pull request by Jim8y**
This pr mainly fix the push bool issue, previously, it pushes 1 or 0 and then convert it to boolean, but actually we can directly use PUSHT and PUSHF.

### CODE REVIEW

File: src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.cs [hunk 1]
```diff
@@ -22,8 +22,8 @@ partial class MethodConvert
 {
     private void Push(bool value)
     {
-        AddInstruction(value ? OpCode.PUSH1 : OpCode.PUSH0);
```
Comment (Jim8y) [inline src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.cs:25]: @shargon do you have any idea why this part was like this?

Comment (shargon) [reply src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.cs:25]: It was less optimal, but should work exactly the same

File: src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.cs [hunk 2]
```diff
@@ -22,8 +22,8 @@ partial class MethodConvert
...
-        ChangeType(VM.Types.StackItemType.Boolean);
+        AddInstruction(value ? OpCode.PUSHT : OpCode.PUSHF);
+        // ChangeType(VM.Types.StackItemType.Boolean);
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.cs:26]:
```suggestion
```

**Review by shargon — APPROVED** [review]

**Review by cschuchardt88 — APPROVED** [review]

**Comment by steven1227** [comment]
Awesome fix!

**Pull request merged**